client/core/trade_simnet_test.go

```diff
@@ -1492,9 +1492,8 @@ func (client *tClient) disableWallets() {
 	// NOTE: this is not reversible, but could be made so with the undo data:
 	// walletPasses[cid] = make(map[uint32]passes, len(client.core.wallets))
 	for _, wallet := range client.core.wallets {
+		wallet.setEncPW([]byte{0})
```

Comment:
This change wasn't necessary since the mutex is locked right below to update `pw`

Reply:
true!